src/types/video-ref.ts

```diff
@@ -0,0 +1,21 @@
+import type {RefObject} from 'react';
+
+export type VideoSaveData = {
+  uri: string;
+};
+
+export interface VideoRef {
+  seek: (time: number, tolerance?: number) => void;
+  resume: () => void;
+  pause: () => void;
+  presentFullscreenPlayer: () => void;
+  dismissFullscreenPlayer: () => void;
+  restoreUserInterfaceForPictureInPictureStopCompleted: (
+    restore: boolean,
+  ) => void;
+  save: (options: object) => Promise<VideoSaveData>;
+  setVolume: (volume: number) => void;
+  getCurrentPosition: () => Promise<number>;
+  setFullScreen: (fullScreen: boolean) => void;
+  nativeHtmlRef?: RefObject<HTMLVideoElement>; // web only
```

Comment:
This would be needed for a user to add custom HTML5 libs (like hls.js, video.js or better subtitles handling).

Reply:
I did not document this, if we go into integrating vifdeo.js/shaka/something this would not be needed anymore and then removed.

Reply:
@zoriya Is it a good idea to expose nativeHtmlRef to the UI ? do it bring some additionnal feature .
maybe you can rename it to nativeHtmlVideoRef it will allow to create new values for shaka / video.js, ...

Reply:
Not sure where to add documentation for this, maybe in the method page? Since it's the page that talk about the video ref?

Reply:
I added it on the method page